apps/studio/src/pages/sites/[siteId]/settings/agency.tsx

```diff
@@ -37,6 +36,12 @@ const AgencySettingsPage: NextPageWithLayout = () => {
   })
   const trpcUtils = trpc.useUtils()
   const toast = useToast()
+  const agencySettingsSchema = Type.Pick(
+    SiteConfigSchema,
+    Type.Union([Type.Literal("siteName"), Type.Literal("agencyName")]),
```

Comment:
question: not sure if the `Type.Union` works but seems like we can just directly do `['siteName', 'agencyName']`?